tests/unit/evalai/test_urls.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth.tokens import default_token_generator
+from django.urls import reverse_lazy
+from django.utils.http import urlsafe_base64_encode
+from django.utils.encoding import force_bytes
+
+from rest_framework.test import APITestCase, APIClient
+
+from django.contrib.auth.models import User
```

Comment:
can you please order these imports alphabetically ?
auth.models import should be at the top